app/api/schema/tickets.py

```diff
@@ -74,6 +79,15 @@ def validate_quantity(self, data):
                                many=True,
                                type_='ticket-tag')
 
+    discount_code = Relationship(
+        attribute='discount_code',
+        self_view='v1.ticket_discount_code',
+        self_view_kwargs={'id': '<id>'},
+        related_view='v1.discount_code_detail',
+        related_view_kwargs={'ticket_id': '<id>'},
+        schema='DiscountCodeSchemaTicket',
+        type_='discount_code')
```

Comment:
A discount code can be applied to multiple tickets and a tickets may have multiple associated discount codes.